### PR TITLE
Server init

### PR DESCRIPTION
**Pull request by davit-y**
run.py has all the endpoints and documentation explaining their use

I will eventually remove venv and replace it with something more compact, but decided to leave it for now to not waste time (also allows for quicker dev setup)

### CODE REVIEW

File: server/tests/audio.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from pydub import AudioSegment
+from pydub.playback import play
+import os
+
+data_dir = os.path.join(os.path.dirname(__file__), 'sounds')
+path = os.path.join(data_dir, 'prototype_loop9.mp3')
+
+sound = AudioSegment.from_file(path, format="mp3")
```
Comment (S-Kantor) [inline server/tests/audio.py]: You might want to change this to mp4, but I think mp3 is still fine.

Comment (davit-y) [reply server/tests/audio.py]: I was just testing the playback of our loops here.

File: server/room.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+class Room:
+    # creator: User
+    # guests: {}
+
+    id: uuid.UUID
+    recording: bool
+    music_producer: MusicProducer
+
+    def __init__(self):
+        self.id = uuid.uuid4()
```
Comment (rohan-dave) [inline server/room.py]: Will this UUID be small enough for users to easily type in? Can we limit the character length to 4 or 5?

Comment (davit-y) [reply server/room.py]: I had this in mind, but was more of an optimization so I decided Ill do it in another PR

**Comment by hmtinc** [comment]
I'm merging this. Commit all future changes to master.

**Pull request merged**